web/app/config/environment.d.ts

```diff
@@ -23,6 +23,7 @@ declare const config: {
   shortLinkBaseURL: string;
```

Comment:
I think we should try to keep `shortLinkBaseURL` consistent everywhere including tests.